Makefile

```diff
@@ -0,0 +1,84 @@
+.DEFAULT_GOAL:=help
+
+BINARY?=opcap
+IMAGE_BUILDER?=podman
+IMAGE_REPO?=quay.io/opdev
+VERSION=$(shell git rev-parse HEAD)
+RELEASE_TAG ?= "0.0.0"
+
+PLATFORMS=linux
+ARCHITECTURES=amd64 arm64 ppc64le s390x
+
+.PHONY: build
+build:
+	go build -o $(BINARY) -ldflags "-X github.com/opdev/opcap/version.commit=$(VERSION) -X github.com/opdev/opcap/version.version=$(RELEASE_TAG)" main.go
+	@ls | grep -e '^opcap$$' &> /dev/null
+
+.PHONY: build-multi-arch
+build-multi-arch: $(addprefix build-linux-,$(ARCHITECTURES))
+
+define ARCHITECTURE_template
+.PHONY: build-linux-$(1)
+build-linux-$(1):
+	GOOS=linux GOARCH=$(1) go build -o $(BINARY)-linux-$(1) -ldflags "-X github.com/opdev/opcap/version.commit=$(VERSION) \
+				-X github.com/opdev/opcap/version.version=$(RELEASE_TAG)" main.go
+endef
+
+$(foreach arch,$(ARCHITECTURES),$(eval $(call ARCHITECTURE_template,$(arch))))
+
+.PHONY: fmt
+fmt: gofumpt
+	${GOFUMPT} -l -w .
+	git diff --exit-code
```

Comment:
Much needed and appreciated! Could you please add another commit on this PR or on a separate one with the changes this target makes? There are only a few changes and mostly improving readability.

Reply:
gofumpt left a pretty significant error in #81 so I am not entirely comfortable with it being here. if it were just gofmt I would be ok with it.